website/app/styles/doc-components/algolia-search/modal.scss

```diff
@@ -0,0 +1,250 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// ALGOLIA AUTOCOMPLETE SEARCH > MODAL
+
+@use "../../breakpoints" as breakpoint;
+@use "../../typography/mixins" as *;
+
+// This class is applied to the `<body>` to remove the scroll when the modal is opened
+.aa-Detached {
+  height: 100vh;
+  overflow: hidden;
+}
+
+
+// OVERLAY (used to apply an opaque background and _contain_ the modal container)
+
+.aa-DetachedOverlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: calc(9999 - 1);
+  height: 100vh;
+  margin: 0;
+  padding: 0;
+  background-color: rgba(#000, 0.5);
+}
+
+// MODAL (container)
+
+.aa-DetachedContainer {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: 9999;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-height: 100vh;
+  margin: 0;
+  overflow: hidden;
+  background: var(--doc-color-white);
+
+  @include breakpoint.with-fixed-sidebar () {
+    position: absolute;
+    top: 90px;
+    width: auto;
+    max-width: 680px;
+    height: auto;
+    margin: 0 auto;
+    border-radius: 4px;
+  }
+}
+
+
+// PANEL (container of "sources")
+
+.aa-Panel {
+  flex-grow: 1;
+}
+
+.aa-PanelLayout {
+  width: 100%;
+  height: auto;
+  max-height: calc(100vh - 300px);
+  margin: 0;
+  padding: 0 16px;
+  overflow-y: auto;
+
+  &:empty {
+    display: none;
+  }
+}
+
+// SOURCES (search results "groups")
+
+.aa-PanelSources {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+
+  &:empty {
+    display: none;
+  }
+}
+
+.aa-Source {
+  padding: 8px 0;
+  border-bottom: 1px solid var(--doc-color-gray-400);
+  transition: opacity 200ms ease-in, filter 200ms ease-in;
+
+  &:last-child {
+    border-bottom: none;
+  }
+
+  // when a request isn't resolved yet
+  .aa-Panel--stalled & {
+    opacity: 0.8;
+    filter: grayscale(1);
+  }
+
+  @media screen and (prefers-reduced-motion) {
+    transition: none;
+  }
+}
+
+// SOURCE > HEADER/FOOTER
+
+.aa-SourceHeader {
+  margin-bottom: 4px;
+}
+
+.aa-SourceHeaderTitle {
+  @include doc-font-style-body-small();
+  color: var(--doc-color-black);
+  font-weight: 600;
+}
+
+
+// not used - could be used for a "show all" button in the future
+// .aa-SourceFooter {}
+
+
+// PANEL FOOTER
+
+.aa-PanelFooter {
+  position: relative;
+  z-index: 9999;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  padding: 16px;
+  background-color: #fff;
+  box-shadow: inset 0 1px 0 rgba(#807ea3, 30%);
+
+  &::after {
+    position: absolute;
+    top: -16px;
```

Comment:
I prefer to use `bottom: 100%;` for positioning items like this. That way if the height were ever adjusted for some reason, you don't have to remember to change a value in 2 places.

Reply:
You're absolutely right. Fixed! 🙏